packages/ag-charts-enterprise/src/series/radial-gauge/radialGaugeSeriesProperties.ts

```diff
@@ -180,6 +200,38 @@ class RadialGaugeBarProperties extends BaseProperties {
 
     @Property
     lineDashOffset: number = 0;
+
+    getStyle(
+        defaultColorRange: string[],
+        scale: _ModuleSupport.LinearScale
+    ): RequireOptional<AgRadialGaugeSeriesStyle> {
+        const {
+            enabled,
+            fill,
+            fills,
+            fillMode,
+            fillOpacity,
+            stroke,
+            strokeWidth,
+            strokeOpacity,
+            lineDash,
+            lineDashOffset,
+        } = this;
+
+        const barFill = enabled
+            ? fill ?? createConicGradient(fills, fillMode, defaultColorRange, scale)
+            : 'rgba(0,0,0,0)';
```

Comment:
Minor - if we use this

```suggestion
            : 'none';
```

it won't try and render the shape